pkg/reconciler/broker/resources/ingress.go

```diff
@@ -30,6 +30,7 @@ import (
 	eventingv1 "knative.dev/eventing/pkg/apis/eventing/v1"
 	reconcilersource "knative.dev/eventing/pkg/reconciler/source"
 	"knative.dev/pkg/kmeta"
+	"knative.dev/pkg/ptr"
```

Comment:
I wonder if at some point we should deprecate this and use this, looks like they added it and does the same thing?
https://pkg.go.dev/k8s.io/utils/pointer

Reply:
I looked into it just now, and k8s pointer doesn't have an equivalent to `ptr.Time`, which [has some usages](https://cs.github.com/?scopeName=All+repos&scope=&q=ptr.Time+knative.dev%2Fpkg%2Fptr) we'll have to remove first. 😕

Reply:
Thanks @imjasonh yeah, wasn't supposed to hold anything up, just popped into my head while looking at this!